.github/workflows/build.yml

```diff
@@ -158,7 +158,7 @@ jobs:
     needs: check_source
     if: needs.check_source.outputs.run_tests == 'true'
     env:
-      OPENSSL_VER: 1.1.1v
+      OPENSSL_VER: 3.0.11
```

Comment:
It looks like we may need to backport c92b391d and/or 576803d7 to be able to do this (and pass CI).